dspace-server-webapp/src/test/java/org/dspace/app/rest/CommunityAdminGroupRestControllerIT.java

```diff
@@ -591,7 +583,9 @@ public void communityAdminRemoveChildGroupFromCommunityAdminGroupPropertySetToFa
     public void communityAdminAddChildGroupToCollectionAdminGroupSuccess() throws Exception {
 
         context.turnOffAuthorisationSystem();
-        Group adminGroup = collectionService.createAdministrators(context, collection);
+        // TODO: Why is this test in CommunityAdmin? it seems to purely be a collection group test?
```

Comment:
Same here, I agree with this TODO (and the other one below this).  This seems to be a copy/paste issue.  I *think* it probably is supposed to test a subgroup of Community Admin (not Collection Admin).   Again though, we could clean this up in a follow-up PR